tokio-macros/src/lib.rs

```diff
@@ -202,6 +202,44 @@ use proc_macro::TokenStream;
 ///         })
 /// }
 /// ```
+///
+/// ### Configure unhandled panic behavior
+///
+/// Available options are `shutdown_runtime` and `ignore`. For more details, see
+/// [`Builder::unhandled_panic`].
+///
+/// ```ignore
+/// #[tokio::main(unhandled_panic = "shutdown_runtime")]
```

Comment:
Since unhandled_panic on multi thread runtime is not implemented yet, so for now it would be better to specify the runtime flavor as well.

```suggestion
/// #[tokio::main(flavor = "current_thread", unhandled_panic = "shutdown_runtime")]
```

Reply:
The macro should probably emit an error if you try to use it with the multi-thread runtime.

Reply:
> The macro should probably emit an error if you try to use it with the multi-thread runtime.

 Thanks for your advice, set `unhandled_panic` behavior on `multi_thread` runtime results in a compile error now.